scripts/ci-k8s-unit-test.sh

```diff
@@ -206,7 +206,10 @@ echo "Install container features in VM"
 run_remote_cmd ${VM_PUB_IP} ${SSH_KEY_FILE} "powershell.exe -command { Install-WindowsFeature -Name 'Containers' -Restart }"
 wait_for_vm_restart
 
-if [ "${JOB_TYPE}" == "presubmit" ]
+set +e  # Temporarily disable errexit
```

Comment:
One alternative to this is to wrap the commands you are checking for an exit code and handling that.

Reply:
I think i tried that but the trap on line 30 was triggering and preventing log collection from running and the junit files weren't being included in the artifacts / job output.

Are you OK keeping this behavior for now?
We can iterate in a separate PR

Reply:
Of course. No issues at all.